scripting/cwx.sp

```diff
@@ -358,6 +368,10 @@ public void OnClientCommandKeyValues_Post(int client, KeyValues kv) {
  * Saves the current item into the loadout for the specified class.
  */
 bool SetClientCustomLoadoutItem(int client, int playerClass, const char[] itemuid) {
+	if (!sm_cwx_allow.BoolValue) {
```

Comment:
Localizations aside, thinking about moving this message to the loadout radio menu, or reworking it so the notice is in the item selection menu (so the notification will be shown when users open it) instead.  Thoughts?  I'm leaning towards the latter myself.  Something like this:

![image](https://user-images.githubusercontent.com/6760398/117692617-0bf62180-b172-11eb-8ffe-d9d16b581c15.png)

Future work for the API possibly involves calling this function; it's not intended to handle things related to the UI.